index.js

```diff
@@ -0,0 +1,5 @@
+import GitHubLinkExtension from './src/components/GitHubLinkExtension';
```

Comment:
Should we also export the Components and MenuComponent ? Better naming might be useful

Reply:
Exporting the template makes sense. I'm making changes to not export extensions and the github extension in favor of building it with app config. I don't want to export components because no one should consume it outside of terra-dev-site.

Reply:
I'm going to leave exporting the template up to @Blackop778 with his work on upgrading the template.